### ISSUE TITLE
New Feature: Add Scheduler

### ISSUE THREAD

**Issue by MichaelFoss** [bhaller/Jiggler#20]
Add a scheduler that whitelists / blacklists when Jiggler can / can't jiggle.

While a little messy, a blacklisting solution is to provide a checkbox with the label "Do not run Jiggler during these times" and have an array of time ranges for specific days of the week (e.g. "Saturday all day" or "Wednesday from 5:00PM to 11:59PM").

Alternatively, a whitelisting solution could be to provide a similar checkbox with the label "Only jiggle during these times" and have the same array of ranges for specific days (e.g. "Monday 9:00AM to 5:00PM").

Either of these solutions would allow Jiggler to run as it does today when they are not checked, but would not jiggle during some times when they are.

**Comment by bhaller** [comment]
Yes, it would be a nice feature.  The user interface for it seems extremely complex, and would require quite a lot of work.  At present I don't have the cycles for that.  If somebody did a high-quality pull request implementing it, that would be great.

**Comment by allanjackson** [comment]
Yes! This is the feature I came to request. Seems like at the simplest, the UI could be a whitelist/blacklist radio button and a time range picker.
